css/index.css

```diff
@@ -75,19 +77,22 @@ p {
     display: flex;
     flex-wrap: wrap;
     justify-content: space-evenly;
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+    
+
 }
 
 .top-content .text-container {
     width: 48%;
```

Comment:
good that you aren't hard coding widths. Percentage, like you used, is the way to go.